<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parser</class>
 <widget class="QMainWindow" name="Parser">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushChromiumWebSerf">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Парсинг js only</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>223</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushParseTarget">
         <property name="text">
          <string>Парсить сайт полностью</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushAddSetting">
       <property name="text">
        <string>Добавить настройку</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pBCustomRegExp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom reg exp</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuGlobal_setting">
     <property name="title">
      <string>Global setting</string>
     </property>
     <widget class="QMenu" name="action_query_parameters">
      <property name="title">
       <string>Параметры запросов</string>
      </property>
      <addaction name="action_cookie"/>
     </widget>
     <addaction name="action_query_parameters"/>
    </widget>
    <addaction name="menuGlobal_setting"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_cookie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Принимать cookie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
